pkg/cli/cli.go

```diff
@@ -97,15 +100,16 @@ func Run(configPath string, inputPath string, outputPath string, deleteOutputFol
 	}
 
 	report := walker.GenerateReport()
-	rPath := filepath.Join(outputPath, reportFileName)
-	reportFile, err := os.Create(rPath)
+	// we store the report in the working dir of the CLI, it should never be created in the output folder
```

Comment:
This comment describes the change being made is not actually useful when read by itself.

Reply:
Also if you want to store it in another location then use a flag which the user can set.

Reply:
it was more intended to be a warning for others changing. But it totally makes sense to add a flag for that, will do later.

Reply:
done